density/templates/predict.html

```diff
@@ -9,20 +9,98 @@
 
 <div class="container">
   <div class="col-12"  style="text-align:center;">
-    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today: </h4>
+    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today.<br><br></h4>
```

Comment:
If we're going to use `<br/>`, then these should be `<br/>` and should probably outside the `<h4>`. 

Theoretically, though, this should be done via CSS styling